app/src/main/java/com/example/demouniverload/search/view/StartActivity.java

```diff
@@ -0,0 +1,29 @@
+package com.example.demouniverload.search.view;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.widget.Button;
+import android.widget.Toast;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.example.demouniverload.R;
+
+public class StartActivity extends AppCompatActivity {
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.start_activity);
+
+        Button startButton = findViewById(R.id.button_start);
+        startButton.setOnClickListener(v -> {
```

Comment:
Can be united in a single line:
```suggestion
        findViewById(R.id.button_start).setOnClickListener(v -> {
```
The same for below.